Cargo.toml

```diff
@@ -8,6 +8,10 @@ codegen-units = 1
 lto = "fat"
 panic = "abort"
 
+[profile.profiling]
```

Comment:
While we're at it let's `strip = true` in `release` and `strip = false` here to match `rise` and Reth.